node/task.ts

```diff
@@ -702,6 +702,24 @@ export function getPlatform(): Platform {
     }
 }
 
+/**
+ * Resolves major version of Node.js engine used by the agent.
+ * @returns {Number} Node's major version.
+ */
+export function getNodeMajorVersion(): Number {
+    const version = process.versions.node;
+    if (!version) {
```

Comment:
When can it be undefined? As I know it's possible in browser/other js runtimes.
Should we just throw in this case?